executor/src/witgen/affine_expression.rs

```diff
@@ -133,14 +133,12 @@ where
     /// Incorporates the case where the symbolic variable `key` is assigned
     /// the value `value`.
     pub fn assign(&mut self, key: K, value: T) {
-        let mut offset = 0.into();
```

Comment:
@chriseth Was there a reason you did it like this in #540, rather than mutating `self.offset` directly?

Reply:
I guess it was the refactoring with the smallest change.